sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/ServiceBusMessageSerializer.java

```diff
@@ -87,9 +87,11 @@ public int getSize(Message amqpMessage) {
         int payloadSize = getPayloadSize(amqpMessage);
 
         final MessageAnnotations messageAnnotations = amqpMessage.getMessageAnnotations();
+        final DeliveryAnnotations deliveryAnnotations = amqpMessage.getDeliveryAnnotations();
```

Comment:
Changelog?

Even though I haven't seen anyone use delivery annotations in EH ... Can we copy this logic over there?

Reply:
> Even though I haven't seen anyone use delivery annotations in EH ... Can we copy this logic over there? (Iirc we allow them to access the raw AMQP message so it could be possible)

Good point, will update EventHubSerializer::getSize as well